custom/src/main/java/com/splunk/opentelemetry/SplunkConfiguration.java

```diff
@@ -41,6 +41,7 @@ public class SplunkConfiguration implements AutoConfigurationCustomizerProvider
   public static final String METRICS_ENDPOINT_PROPERTY = "splunk.metrics.endpoint";
   public static final String METRICS_EXPORT_INTERVAL_PROPERTY = "splunk.metrics.export.interval";
   public static final String METRICS_IMPLEMENTATION = "splunk.metrics.implementation";
+  public static final String METRICS_FULL_COMMAND_LINE = "splunk.metrics.force_full_commandline";
```

Comment:
Usually we use dashes instead of underscores in properties' names:
```suggestion
  public static final String METRICS_FULL_COMMAND_LINE = "splunk.metrics.force-full-commandline";
```

Reply:
Will you allow me to break with tradition here in the name of making things better? I think the hyphens are actually problematic because they don't translate cleanly to environment vars. We've done a pretty good job of documenting both, but it definitely still opens up an opportunity for users to start with or copy the system property and manually convert it to env var by uppercasing and changing periods to underscores, which WILL NOT WORK if the original system property contains hyphens.

I dunno, I don't feel that strongly about it and can err on the side of consistency, even if it's worse. 😁

Reply:
What do you mean `handle underscores too`? If they're already underscores, what would need to change?

Reply:
Like, you could pass either `splunk.metrics.force-full-commandline` or `splunk.metrics.force_full_commandline` as a system property and both would work the same way

Reply:
[Is that not true today](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/ConfigPropertiesUtil.java#L32)? I thought it was basically just "dots and hyphens to underscores, then uppercase".

Reply:
Nope: https://github.com/open-telemetry/opentelemetry-java/blob/727850e94faa1edcb45c469620e7fa1f9baecf60/api/all/src/main/java/io/opentelemetry/api/internal/ConfigUtil.java#L61

`ConfigPropertiesUtil` is only used in very few cases where the SDK config is not available.

Reply:
Ah, thanks for clarifying! That separation between `Config` and `ConfigProperties` always seemed questionable to me. :) I was thinking more of the interop between properties and env vars, and less about normalizing among system property names.